althea_types/src/interop.rs

```diff
@@ -146,6 +146,7 @@ impl Message for Identity {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
 pub struct LocalIdentity {
     pub wg_port: u16,
+    pub have_tunnel: Option<bool>, // If we have an existing tunnel, None if we don't know
```

Comment:
Maybe there is some more information we can put here instead of just bool?

Reply:
A bool is all we really need, although the option + bool setup really makes it the same as two bools.